code/src/components/Footer.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import styled from 'styled-components';
+
+// -----------------------------------------------------------------------------
+
+export const Footer = () => {
+  // Sneaky hidden function to clear local storage
```

Comment:
Awesome done! Will use this in my own code 😃

Reply:
Haha this caused me to accidentally remove my todos 🙈